views/table.html

```diff
@@ -3,7 +3,11 @@
   <h3 class="s-q-item__title">{{ item.title }}</h3>{% endif %} {%- if item.subtitle and item.subtitle !== '' %}
   <div class="s-q-item__subtitle">{{ item.subtitle }}</div>
   {%- endif %}
-  <div style="overflow-x: auto">
+  <div style="overflow-x: auto;padding: 0 1px;">
```

Comment:
What's the `padding` for? I tried to remove it, haven't seen any difference or uses to it.

Reply:
Yes, exactly. The `padding: 0 1px;` is more of a workaround, than a final solution.